messaging/src/main/java/org/axonframework/lifecycle/Phase.java

```diff
@@ -74,4 +74,8 @@ private Phase() {
      * Phase to start or shutdown components which send/receive/handle instructions for/to other connectors.
      */
     public static final int INSTRUCTION_COMPONENTS = INBOUND_EVENT_CONNECTORS + 10;
+    /**
+     * Phase to start or shutdown event stores.
+     */
+    public static final int EVENT_STORE = INBOUND_EVENT_CONNECTORS - 10;
```

Comment:
I'd rather have the `EmbeddedEventStore` directly use the phase `INBOUND_EVENT_CONNECTORS - 10` than adding a new one. Stating this, as the chances are slim other `EventStore` implementations or other Axon infrastructure components will need to tie into this phase.

Reply:
Made the suggested change.